js/src/components/contact-information/phone-number-card/phone-number-card.js

```diff
@@ -117,14 +117,27 @@ const PhoneNumberCard = ( {
 } ) => {
 	const { loaded, data } = phoneNumber;
 	const [ isEditing, setEditing ] = useState( initEditing );
+	const onPhoneNumberVerifiedRef = useRef();
```

Comment:
❓  What is the advantage of useRef vs useCallback?

(I take as granted this is for avoiding error as `onPhoneNumberVerified` needs to be a dependency in `useEffect`

Reply:
> What is the advantage of useRef vs useCallback?

I think it's probably not a choice about advantages, but rather what the concern is to be addressed here. `useCallback` is used when needing a callback with stable reference based on deps array. `useRef` has plenty of use scenarios and one of them is the need of ignoring the reference changes such as the use case here.

> I take as granted this is for avoiding error as `onPhoneNumberVerified` needs to be a dependency in `useEffect`

Yes, it avoids the possibility of infinity re-rendering loop.